plane/source/docs/quadplane-parameters.rst

```diff
@@ -29,6 +29,37 @@ Key Parameters
    throttle while flying or you risk disarming your motors.
 -  The default :ref:`SCHED_LOOP_RATE<SCHED_LOOP_RATE>` for a QuadPlane is to 300 (Hz). Most QuadPlanes do not need this to be rasied. Some very small vehicles (< 1Kg) might benefit from setting it to 400. In heavier vehicles, their higher inertia results in lower effective control response rates, so they do not benefit from a higher loop rate. Raising above 300 only leads to larger log files in these vehicles.
 
+.. _return_behavior_setup:
+
+Return Behavior QuickSetup Guide:
```

Comment:
This seems like a funny place to put this on the bottom of a page that is generally about Q parameters. Perhaps it should be on it's own page.

Reply:
I disagree...this is basic QuadPlane setup....it is not related to the other topics like frame selection, etc in this major First Time Setup group....if the page expands more, then putting the other sections along with this, on individual pages _**might**_ be warranted, but that may never happen